CONTRIBUTING.rst

```diff
@@ -39,9 +39,11 @@ generate pull requests to submit your code for inclusion in the project.
 .. _GitHub help for collaborating on projects using issues and pull requests:
    https://help.github.com/categories/collaborating-on-projects-using-issues-and-pull-requests/
 
-All pull requests will be automatically tested as described below, so make sure
-you test them yourself before submitting them::
+All pull requests will be automatically tested as described below using the
+``pr-check.sh`` script, so make sure you test them yourself before submitting
+them::
 
+  # do not run this if using Anaconda; instead, look at pr-check.sh
```

Comment:
Can you also elaborate why running this via Anaconda is not recommended? Using Anaconda one can directly run `./pr-check.sh` to test out before making the PR.